test/e2e/providers/aws/aws.go

```diff
@@ -390,6 +390,7 @@ func (a *awsProvider) GenerateMachineSet(withWindowsLabel bool, replicas int32)
 			Template: mapi.MachineTemplateSpec{
 				ObjectMeta: mapi.ObjectMeta{Labels: machineLabels},
 				Spec: mapi.MachineSpec{
+					ObjectMeta: mapi.ObjectMeta{Labels: map[string]string{"node-role.kubernetes.io/worker": ""}},
```

Comment:
Is this needed? I thought having the `machine.openshift.io/cluster-api-machine-*` labels will result in this label being applied to the `Machine`.  If that is not the case then this too should be called out in the README. Just having it in the example is not sufficient.

Reply:
That label would go to the machine object created. `node-role.kubernetes.io/worker: ""` should applied in the machine spec so that the label will go on to the node object created from the machine object. However I added it to the list of required labels for the machineset.